app/assets/stylesheets/_bootstrap-custom.scss

```diff
@@ -50,3 +50,8 @@ $enable-negative-margins: true;
 // Utilities
 @import "bootstrap/utilities/api";
 // scss-docs-end import-stack
+
+// TODO: move this somewhere better
+.section:nth-child(odd) {
```

Comment:
I'll figure out a better place to put this once all of the old CSS has been deleted.